ansible/local_testing/Vagrantfile

```diff
@@ -2,19 +2,20 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "bento/debian-11"
-    config.vm.box_version = "202212.11.0"
+    config.vm.box = "bento/debian-12.5"
+    config.vm.box_version = "202404.23.0"
     config.vm.provision "shell", inline: <<-SHELL
-        apt-get update
+        sudo apt-get update
         apt-get install -y python3 openssh-server
+        apt-get install -y python3 ansible
         systemctl enable ssh
     SHELL
 
     config.vm.define "control", primary: true do |control|
         control.vm.hostname = "control"
         control.vm.network "private_network", ip: "192.168.56.1",
             virtualbox__intnet: true
-        control.vm.synced_folder "../", "/home/vagrant/infra"
+        control.vm.synced_folder "../", "/home/vagrant/infra", type: "rsync"
```

Comment:
Does the regular synced folder not work here?

Reply:
It wasn't syncing the folders for me, I ended up with an emtpy /home/vagrant directory

Reply:
Having looked at the docs, it says vagrant chooses the best fitting sync type based on your setup.

I think it doesn't hurt to set an explicit sync type here better to rely on something we do not see nor have control over

Reply:
Sounds good to me. Strange that it doesn't work properly though. Perhaps once again a :sparkles: Windows speciality :fireworks: